android/app/src/main/kotlin/com/cBreez/client/MainActivity.kt

```diff
@@ -1,6 +1,27 @@
 package com.cBreez.client
 
+import android.os.Bundle
 import io.flutter.embedding.android.FlutterActivity
+import io.flutter.util.PathUtils
+import org.tinylog.kotlin.Logger
+import java.io.File
 
-class MainActivity: FlutterActivity() {
+class MainActivity : FlutterActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        configureLogger()
```

Comment:
Is the Activity created when the app is killed and the notification is received?